docs/resources/blocks-and-transactions.md

```diff
@@ -0,0 +1,1560 @@
+            "value": "1",
+            "index": true
+          },
+          {
+            "key": "proposal_result",
+            "value": "proposal_passed",
+            "index": true
+          }
+        ]
+      }
+    ],
+    "validator_updates": null,
+    "consensus_param_updates": {
+	  ...
+    }
+  }
+}
+```
+
+### 2. MsgDeposit
+
+- **Descriptions:** : Submit a deposit to an existing proposal
+
+    Deposit will transfer from the proposer account to the "gov" module account.
+
+<!-- omit in toc -->
+
+#### Protobuf
+
+```go
+type MsgDeposit struct {
+	ProposalId uint64                                   `protobuf:"varint,1,opt,name=proposal_id,json=proposalId,proto3" json:"proposal_id" yaml:"proposal_id"`
+	Depositor  string                                   `protobuf:"bytes,2,opt,name=depositor,proto3" json:"depositor,omitempty"`
+	Amount     github_com_cosmos_cosmos_sdk_types.Coins `protobuf:"bytes,3,rep,name=amount,proto3,castrepeated=github.com/cosmos/cosmos-sdk/types.Coins" json:"amount"`
+}
+```
+
+<!-- omit in toc -->
+
+#### Example
+
+Cosmos Transaction Query API: [https://mainnet.crypto.org:1317/cosmos/tx/v1beta1/txs/9CCC988616344C804E8831B6FC6BECD6FD0F815E4E3FF13BDE6B7F8360BF0050](https://mainnet.crypto.org:1317/cosmos/tx/v1beta1/txs/9CCC988616344C804E8831B6FC6BECD6FD0F815E4E3FF13BDE6B7F8360BF0050)
+
+<!-- omit in toc -->
+
+#### Details
+
+| Detail               | Accessor                                                                        | Type                        |
+| -------------------- | ------------------------------------------------------------------------------- | --------------------------- |
+| Transaction Type     | `tx.body.messages[message_index]["@type"] === "/cosmos.gov.v1beta1.MsgDeposit"` | String                      |
+| Deposit From Address | `tx.body.messages[message_index].depositor`                                     | String                      |
+| Deposit Amount       | `tx.body.messages[message_index].amount`                                        | [Asset Array](#_2-array) |
+
+[Top](#table-of-content)
+
+### 2a. Burn Proposal's Deposit if Proposal does not Get Enough Deposit
+
+If a proposal does not meet the depsoit requirement after the deposit period, the deposit will **NOT** be returned to the depositors. Those deposits **will be burnt** from the "gov" module account as well.
+
+The latest deposit requirement ("min_deposit") and deposit period ("max_deposit_period") can be checked on https://mainnet.crypto.org:1317/cosmos/gov/v1beta1/params/deposit. Note that they are network parameters and may change over time after governance proposals.
+
+To monitor a proposal becomes inacitve, it can be detected by monitoring the `end_block_events` in Tendermint Block Results API. However, for the amount of deposit burnt, you have to keep track of the deposits made to the proposal before. Note that this operation does not involve any user account as the deposits are burnt.
```

Comment:
```suggestion
To monitor a proposal that becomes inactive, it can be detected by monitoring the `end_block_events` in Tendermint Block Results API. However, for the amount of deposit burnt, you have to keep track of the deposits made to the proposal before. Note that this operation does not involve any user account as the deposits are burnt.
```